server/src/subscriptions.js

```diff
@@ -1,29 +1,6 @@
-const {
-  execute,
-  subscribe
-} = require('graphql')
 const { PubSub } = require('apollo-server');
-const { SubscriptionServer } = require('subscriptions-transport-ws')
 
-function subscriptionServer (keycloakService, httpServer, apolloServer) {
-    return new SubscriptionServer({
-      execute,
-      subscribe,
-      onConnect: async connectionParams => {
-        if(keycloakService) {
-          return await keycloakService.validateToken(connectionParams)
-        } else {
-          return true;
-        }
-      },
-      schema: apolloServer.schema
-    }, {
-      server: httpServer,
-      path: '/graphql'
-    });
-}
 
 module.exports = {
-    pubSub: new PubSub(),
-    subscriptionServer
+  pubSub: new PubSub()
```

Comment:
I think we should just move this to the subscriptions schema file in tasks if its the only thing this file is being used for. thats the only place it is used so a one liner there makes sense.

Reply:
+1 on that point.